apps/ledger-live-desktop/src/main/internal-lifecycle.js

```diff
@@ -228,4 +225,6 @@ internalHandlerPromise(transportOpenChannel);
 internalHandlerPromise(transportExchangeChannel);
 internalHandlerPromise(transportCloseChannel);
 internalHandlerObservable(transportExchangeBulkChannel);
+internalHandlerObservable(transportListenChannel);
 internalHandlerEvent(transportExchangeBulkUnsubscribeChannel);
+internalHandlerEvent(transportListenUnsubscribeChannel);
```

Comment:
It could have been interesting to have a small documentation (if there is one already to link it here) of what those handlers are doing and why they are necessary 🙏

Reply:
like: `internal` represents the renderer thread ? It feels like the name is not enough explicit with a quick doc

Reply:
I have better: 3 videos :) https://github.com/LedgerHQ/ledger-live/pull/2342